packages/generator-emakinacee-react/generators/app/templates/shared/dot/.eslintrc

```diff
@@ -1,7 +1,10 @@
 {
     "extends": [
         "emakinacee-base",
-        "emakinacee-react"
+        "emakinacee-react",
+        // Must be last to override rules. https://github.com/prettier/eslint-config-prettier#installation
+        "prettier",
+        "prettier/react"
```

Comment:
These rules do disable rules, which interfere with prettier, but all syntactic errors are still linted (like duplicate imports for example). I tried prettier on the generated App.js file, and it did not change anything. There is another way, but I **think** this would be the easier way to use Prettier. 

There is the approach to pass the prettier formatted rules to `eslint --fix`. That may have its own problems though, I do not know how robust it is. https://github.com/prettier/prettier-eslint